<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedRoi</class>
 <widget class="QWidget" name="advancedRoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>901</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Select Volume of Interest</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>1021</width>
     <height>811</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="fullScreenLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="imFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>321</width>
        <height>301</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QPushButton" name="saveChangesButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
        </property>
        <property name="text">
         <string>Save Changes</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QPushButton" name="revertButton">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
        </property>
        <property name="text">
         <string>Revert To Original ROI</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
